src/Native/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ include_directories("${CMAKE_BINARY_DIR}/../../")
 if(WIN32)
     add_definitions(-DWIN32)
     add_definitions(-D_WIN32=1)
+    add_definitions(-DUNICODE -D_UNICODE)
```

Comment:
Looking at 

https://github.com/dotnet/coreclr/blob/7b41d8d938760c2e5d54d2e699a5bbb78cd3502a/clrdefinitions.cmake#L229-L230
and
https://github.com/dotnet/coreclr/blob/b82063e24b34c875891916a1bbc01728b9022bf3/src/vm/CMakeLists.txt#L7

It looks like these options are set for all platforms in coreclr.  So maybe we should be setting them for all platforms as well?

/cc @janvorli @jkotas - any thoughts?

It doesn't appear that we are setting these options at all in the corefx or core-setup repos.

Reply:
I do not think you want to set these for this project.

CoreCLR is built with Win32 PAL that is a custom Windows API emulator. You will find a lot of Windowsisms in Unix CoreCLR build because of that. `UNICODE` define is one of them.

If you want to look at prior art, CoreRT build is better example. It is setup as clean x-plat project. You can see that it defines `UNICODE` on Windows only: https://github.com/dotnet/corert/blob/master/src/Native/CMakeLists.txt#L185

Reply:
Thanks for the info, @jkotas.